cmd/apply.go

```diff
@@ -13,29 +13,30 @@ import (
 
 var Apply = func() *cli.Command {
 	return &cli.Command{
-		Name:      "apply",
-		Usage:     "Runs an apply with optional auto-approval",
-		UsageText: "nullstone apply [--stack=<stack-name>] --block=<block-name> --env=<env-name> [options]",
+		Name:        "apply",
+		Description: "Runs a Terraform apply on the given block and environment. This is useful for making ad-hoc changes to your infrastructure.\nBe sure to run `nullstone plan` first to see what changes will be made.",
```

Comment:
Great updates to the wording.

I don't know where `Description` appears in the CLI help text, but I imagine this field is helpful for the docs.

For `Usage` (which appears in the `nullstone apply --help`) and `Description`, it would be helpful to identify *where* the plan/apply is happening. In this case, it's running a plan/apply on the nullstone servers. 

Additionally, it may be helpful to refer users to `nullstone workspaces select` if they wish to run plan/apply locally.

Reply:
Below is how the `Description` shows up in the CLI help text.

![Screen Shot 2022-10-24 at 11 38 15 AM](https://user-images.githubusercontent.com/771008/197567219-5ced4dfb-b0a9-4747-ba29-9949611d2991.png)